test/driver.js

```diff
@@ -582,6 +582,16 @@ var Driver = (function DriverClosure() { // eslint-disable-line no-unused-vars
       this._log('Done !');
       this.end.textContent = 'Tests finished. Close this window!';
 
+      var re = new XMLHttpRequest();
+      re.open('POST','/browserTestReports?path=' + escape(this.appPath),false);
```

Comment:
This `path` query parameter is not used anywhere in test.js. Just use `'/browserTestReports'` . And add a space after the commas.

Reply:
I'll make the changes